Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -1491,10 +1491,10 @@ def return_outputs(readable_output, outputs=None, raw_response=None):
     # Return 'readable_output' only if needed
     if readable_output and not outputs and not raw_response:
         demisto.results(readable_output)
```

Comment:
The better solution will be if you change here
```
return_entry["Contents"] = readable_output
return_entry["ContentsFormat"] = formats["text"]
demisto.results(return_entry)
```

Reply:
this way if the test returns "ok" it will set ok to contents.
And in case someone uses the return_outputs to return just markdown to the warroom, the function will return markdown as it suppose to do, instead of just doing demisto.results()